adoc/retail_chap_formulas.adoc

```diff
@@ -0,0 +1,249 @@
+* The [systemitem]``domain-name-servers`` option must point to the branch server for correct host name resolution
+====
+
+
+.Procedure: Setting up a branch server with a shared network
+
+. In the {susemgr} {webui}, open the details page for the branch server, and navigate to the [guimenu]``Formulas`` tab.
+. In the [guimenu]``Branch Network`` section, set these parameters:
+* Keep [guimenu]``Dedicated NIC`` unchecked
+* Select which services to enable on the branch server’s firewall.
+Ensure you include DNS, TFTP and FTP services.
+* Select the [guimenu]``bind`` DNS forwarder mode.
+* Check DNS forwarder fallback if you want to rely on an external DNS if the branch DNS fails.
+* Specify the working directory, and the directory owner and group.
+. Click btn:[Save] to save your changes.
+. Apply the highstate.
+
+
+
+[[retail.sect.formulas.pxe]]
+== PXE Formula
+
+The PXE formula is used to configure PXE booting on the branch server.
+
+
+.Procedure: Configuring PXE booting
+. In the {susemgr} {webui}, open the details page for the branch server, and navigate to the [guimenu]``Formulas`` tab.
+. Select the [systemitem]``Pxe`` formula, and click [btn]``Save``.
+. Navigate to the menu:Formulas[Pxe] tab, and set these parameters:
+* In the [guimenu]``Kernel filename`` field, keep the default value.
+* In the [guimenu]``Initrd filename`` field, keep the default value.
+* In the [guimenu]``Kernel commandline parameters`` field, keep the default value.
+* In the [guimenu]``PXE root directory`` field, enter the path to the saltboot directory (for example, [systemitem]``/srv/saltboot``).
+* In the [guimenu]``Branch id`` field, type a name to use as a branch identifier (for example: [systemitem]``Branch0001``).
+Use only alphanumeric characters for the branch identifier.
+. Click [btn]``Save Formula`` to save your configuration
+. Apply the highstate.
+
+
+[[retail.sect.formulas.saltboot]]
+== Saltboot Formula
+
+The Saltboot formula is used to configure the boot image on a terminal.
+
+[NOTE]
+====
+There are many different configurations possible.
+This procedure outlines a simple, single-disk, three-partition configuration.
+Adjust it to suit your own environment.
+====
+
+
+.Procedure: Configuring the branch server with saltboot
+
+. In the {susemgr} {webui}, open the details page for the branch server, and navigate to the [guimenu]``Formulas`` tab￼.
+. Check the [systemitem]``Bind`` formula, and click [btn]``Save``.
+. Navigate to the menu:Formulas[Bind] tab, and set these parameters for Zone1:
+* In the [guimenu]``Config`` section, select [systemitem]``Include Forwarders``.
+* In the [guimenu]``Name`` field, use the fully qualified domain name (FQDN) of the  branch server (for example: [systemitem]``branch1.mycompany.org``).
+* In the [guimenu]``Type`` field, select [systemitem]``master``.
+. Click [btn]``Add item`` to save your changes.
```

Comment:
Add item does not save anything. It will just create new input fields.

Reply:
This procedure has been removed as per your earlier comment.